docs/examples/federated_learning_example_link.rst

```diff
@@ -0,0 +1 @@
+.. mdinclude:: ../../examples/federated_learning/surface_defect_detection/README.md
```

Comment:
How about just linking the markdown files to omit these extra link files?

Reply:
fix